src/PublishCodeCovCoverageTask/__tests__/webUtils.test.ts

```
@@ -0,0 +1,672 @@
+      test('should abort download when AbortSignal is triggered', async () => {
+        // Create direct mock implementations for WebUtils' downloadFile that simulates abort
+        const originalDownloadFile = require('../utils/webUtils').downloadFile;
+
+        // Replace with a controlled mock that simulates abort behavior
+        const mockDownloadFile = jest.fn().mockImplementation((fileUrl, dest, options = {}) => {
+            if (options.signal) {
+                // Return a promise that is rejected when abortion is detected
+                return new Promise((resolve, reject) => {
+                    // We won't actually use addEventListener here - we'll simulate its behavior
+                    setTimeout(() => {
+                        reject(new Error('Download aborted by user: ' + fileUrl));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)